src/constants.js

```diff
@@ -1,6 +1,6 @@
 export const TMDB_BASE_URL = 'https://api.themoviedb.org/3/';
 export const BASE_IMAGE_URL = "https://image.tmdb.org/t/p/w500";
-export const BASE_LOCAL_URL = "https://entertainment-app-backend-k2wa.onrender.com/";
+export const BASE_LOCAL_URL = "http://localhost:4000/";
```

Comment:
Please push the remote link which we had.

Reply:
done